build/squire-raw.js

```diff
@@ -5190,47 +5192,6 @@ proto.makeUnorderedList = command( 'modifyBlocks', makeUnorderedList );
 proto.makeOrderedList = command( 'modifyBlocks', makeOrderedList );
 proto.removeList = command( 'modifyBlocks', removeList );
 
-// Node.js exports
```

Comment:
Maybe my last email was not clear, but we still need these.  The rakefile has to be updated to match the makefile

Reply:
changed with e98ec3d